docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
   - Product ID: 1
   - Revocation level: 0
 ```
-This is the [remote attestation](enclaves.md#remote-attestation), an object which proves certain information about the
-enclave that has been loaded. The private key that corresponds to the "Session encryption key" (printed above) is only
-available inside the enclave. Our client will use the "Session encryption key" to encrypt data to send into the enclave.
+This output is the [remote attestation](enclaves.md#remote-attestation), an object which proves certain information
+about the enclave. The private key corresponding to the "Session encryption key" (printed above) is only available
+inside the enclave. The client will use the "Session encryption key" to encrypt data to send to the enclave.
+
+After the server starts up, it will be ready to communicate with the client on http://localhost:8080.
 
-Once the server is done starting up it will be ready to communicate with the client on http://localhost:8080.
-You can proceed to [Run the client](#run-the-client) when you see the following output:
+2. Proceed to [Run the client](#run-the-client) when you see the following output:
 
 ```text
-[main] INFO com.r3.conclave.host.web.EnclaveWebHost$Companion - Started EnclaveWebHost.Companion in <SECONDS> seconds 
+[main] c.r.c.host.web.EnclaveWebHost$Companion  : Started EnclaveWebHost.Companion in <SECONDS> seconds 
 ```
 
-!!! warning
-    The sample uses Conclave mail which handles encryption and authentication, but the web host protocol makes use of
-    request headers which are not encrypted. Because of this, it is still important to use HTTPS for anything other than
-    internal development. Setting up an HTTPS connection is beyond the scope of this tutorial, but you may wish to look
-    at configuring Spring Boot or setting up a reverse proxy such as Nginx or Apache. See
-    [Conclave web host](conclave-web-host.md) for more information.
+!!!Warning
+
+    The sample uses Conclave Mail which handles encryption and authentication. But the web host protocol uses
+    unencrypted request headers. So, it is important to use HTTPS for anything other than internal development.
+    Setting up an HTTPS connection is beyond the scope of this tutorial. You can try configuring Spring Boot or 
+    setting up a reverse proxy such as Nginx or Apache. See [Conclave web host](conclave-web-host.md) for more 
+    information.
 
 ## Run the client
 
-Run the client with
+After running the host:
+
+1. Open another command line interface.
```

Comment:
Similarly as above, I would remove first step.

Reply:
Same comment as above.

Reply:
And what about: _After running the host, in another command line interface:_ 
So then we don't need to mention it in the host section.

Reply:
Correct. I will remove it.

Reply:
Many readers will skim the intro note and jump to the steps.

Rewrote as: 

To run the client:
1. Open another command line interface.
2. Go to the hello-world directory and run the client.